nextcloud/src/main/java/ch/cyberduck/core/nextcloud/NextcloudHomeFeature.java

```diff
@@ -37,19 +35,20 @@ public NextcloudHomeFeature(final Host bookmark) {
     }
 
     @Override
-    public Path find() throws BackgroundException {
+    public Path find() {
         return this.find(Context.files);
     }
 
     public Path find(final Context files) {
-        if(StringUtils.isBlank(bookmark.getCredentials().getUsername())) {
+        String username = bookmark.getCredentials().getUsername();
+        if(StringUtils.isBlank(username)) {
             if(log.isWarnEnabled()) {
                 log.warn(String.format("Missing username for %s", bookmark));
             }
             return null;
         }
         final Path workdir = new Path(new Path(String.format("/remote.php/dav/%s", files.name()), EnumSet.of(Path.Type.directory)),
-                URIEncoder.encode(bookmark.getCredentials().getUsername()), EnumSet.of(Path.Type.directory));
+                username, EnumSet.of(Path.Type.directory));
```

Comment:
No need to URI encode `username`?

Reply:
This is handled separately when doing requests but file paths in the model are never encoded.